crmlogin/actualContactEdit.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/*
+Takes in values from contactedit2.php and changes them in the database.
+
+*/
+	include 'Contact.php';
```

Comment:
require instead of include. If the include fails... the rest of this sript is useless.